src/Microsoft.ApplicationInsights.AspNet/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -48,6 +49,9 @@ public static void AddApplicationInsightsTelemetry(this IServiceCollection servi
             services.AddSingleton<ITelemetryInitializer, WebSessionTelemetryInitializer>();
             services.AddSingleton<ITelemetryInitializer, WebUserTelemetryInitializer>();
 
+            var projectJsonConfiguration = new ConfigurationBuilder().AddJsonFile("project.json").Build();
```

Comment:
Configuration is already passed as an argument. You can pass it into initializer constructor